### PR TITLE
Adds dreams for the MOUSE category

### PR DESCRIPTION
**Pull request by ghost**
```SUMMARY: Content "The mouse category of mutations now has proper dreams."```  

#### Purpose of change
This was simply an oversight. Every other mutation category has dreams related to that category, so here's some for mice.

### CODE REVIEW

File: data/json/dreams.json [hunk 1]
```diff
@@ -489,6 +505,14 @@
             ],
         "category" : "RAPTOR",
         "strength" : 3
+    },{
+        "type" : "dream",
+        "messages" : [
+            "You weave between the zombies' legs, too small for them to catch.  The gas station isn't far now, with all its soda and chips and- UGH, dream...",
+            "Once you finish your tasty dinner of seeds, you look around for your drink, but you already ate it."
```
Comment (MT-Arnoldussen) [inline data/json/dreams.json]: You ate the drink?

Comment (DracoGriffin) [reply data/json/dreams.json]: I believe a [reference to acquiring all their hydration needs through eating](https://en.wikipedia.org/wiki/Kangaroo_rat#cite_ref-Lidicker1960_9-0).

Comment (Xhuis) [reply data/json/dreams.json]: It's referencing metabolic rehydration, yeah. Rewording could be a thing.

Comment (MT-Arnoldussen) [reply data/json/dreams.json]: Ah alright. I thought it referenced your lack of impulse control or something.

**Pull request merged**